dpgv4.py

```diff
@@ -343,7 +345,7 @@ def prepare_audio_conversion_command(input_file: str, aid: Optional[int]) -> Seq
         "-f", "data",
         "-map", f"0:a:{0 if aid is None else aid}",
         "-c:a", "mp2",
-        "-b:a", "128k",
+        "-b:a", "320k",
```

Comment:
This change is not acceptable for me because of the output file size increase it entails. Please make it dependent on high quality mode (make the `prepare_audio_conversion_command` function accept a `quality` parameter and set the bitrate accordingly).